content/sensu-core/0.29/guides/pre-compile-plugins.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Pre-Compiling Plugins"
+description: "Pre-Compiling Sensu Plugins for Distribution"
+product: "Sensu Core"
+version: "0.29"
+weight: 2
+next: ../intro-to-checks
+previous: ../overview
+menu:
+  sensu-core-0.29:
+    parent: guides
+---
+
+# Overview
+
+This guide will walk you through an example on how to pre-compile Sensu Plugins for distribution among clients.
+The goal of this guide is to showcase how you can quickly install plugins on ephemeral instances where time to compile can affect time to production  .
+We'll be using `sensu-plugins-aws` as it requires `nokogiri`,  usually requiring a longer build time.
+
+# Prerequist
+
+Sources:
+
+* [GitHub - luislavena/gem-compiler: A RubyGems plugin that generates binary gems](https://github.com/luislavena/gem-compiler)
+
+This guide assumes the following:
+
+* Centos 7 platform
+* You have some way to store these artifacts and make them available for systems in your environment
+
+_NOTE: Compiled gems are tied to the Ruby version they are compiled for. If Sensu upgrade Ruby 2.4 -> Ruby 2.5, packages will need to be rebuilt._
```

Comment:
Maybe do: "If Sensu upgrades Ruby, for example from 2.4 to 2.5, packages will need to be rebuilt."

And if you still like the current way, you'll at least need to change `upgrade` to `upgrades`.